src/rust/aggregator/src/data/user_history.rs

```diff
@@ -137,10 +137,13 @@ impl Data for UserHistory {
         .await
         .map_err(|e| DataAggregationError::ProcessingError(anyhow!(e)))?;
         for x in &limit_events {
+            let txn = x.txn_version.to_bigint().ok_or(DataAggregationError::ProcessingError(anyhow!("txn_version not integer")))? << 64;
+            let event = x.event_idx.to_bigint().ok_or(DataAggregationError::ProcessingError(anyhow!("event_idx not integer")))?;
+            let txn_event: BigDecimal = BigDecimal::from(txn & event);
```

Comment:
@CRBl69 You mean `(txn << 64) | (event)`

Reply:
The `<< 64` is done at line 140.